packages/overlays/stories/index.stories.mdx

```diff
@@ -0,0 +1,646 @@
+import { Story, Meta, html } from '@open-wc/demoing-storybook';
+import { renderLitAsNode } from '@lion/core';
+import {
+  OverlayMixin,
+  withBottomSheetConfig,
+  withDropdownConfig,
+  withModalDialogConfig,
+} from '../index.js';
+
+import './lion-demo-overlay.js';
+import './applyDemoOverlayStyles.js';
+
+<Meta title="Overlay System/System" parameters={{ component: 'lion-demo-overlay' }} />
+
+# Overlays System
+
+Supports different types of overlays like dialogs, toasts, tooltips, dropdown, etc.
+
+Manages their position on the screen relative to other elements, including other overlays.
+
+`lion-dialog` can be used to make it easy to use our Overlay System declaratively. It is a simple Web Component that implements the `OverlayMixin`.
```

Comment:
As discussed, I would recommend people to directly extend OverlayMixin instead of having a layer inbetween that should be overridden